EasyCommands/Common/Utilities.cs

```diff
@@ -28,6 +28,7 @@ partial class Program {
 
         //Utilities for constructing collections with few characters
         public static List<T> NewList<T>(params T[] elements) => new List<T>(elements);
+        public static T[] NewArray<T>(params T[] elements) => elements;
```

Comment:
This function is pointless if it is only used to fill params arguments. just pass the arguments directly.